CRM/Core/BAO/CustomField.php

```diff
@@ -196,9 +196,17 @@ public static function bulkSave($bulkParams, $defaults = []) {
       $sql[$params['table_name']][] = $fieldSQL;
       $customFields[$index] = $customField;
     }
+
+    $config = CRM_Core_Config::singleton();
     foreach ($sql as $tableName => $statements) {
       // CRM-7007: do not i18n-rewrite this query
       CRM_Core_DAO::executeQuery("ALTER TABLE $tableName " . implode(', ', $statements), [], TRUE, NULL, FALSE, FALSE);
+
+      if ($config->logging) {
```

Comment:
just stylistically can we change this to a one liner

CRM_Core_Config::singleton()->logging

Reply:
Done!